docs/docs/communicate/js-from-fable.md

```diff
@@ -484,6 +484,116 @@ myLib.myMethod(Vertical, Horizontal)
 myLib.myMethod("vertical", "Horizontal")
 ```
 
+#### TypeScriptTaggedUnion attribute
+
```

Comment:
Should we add a message saying that this is supported only since Fable version XX?

Reply:
Will the next version be 3.6.2 or 3.7.0?

Reply:
Personally, I would say it should be 3.7.0 but we will have to wait for @alfonsogarciacaro to release it to know :)

Reply:
Yes, it should be 3.7, but for no particular reason I thought it would be nice to align Fable's minor version with F# 6 so I released it as 3.6.2 😅 I will update the docs, thanks @cannorin!